WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmail/LoginViaEmailViewModel.swift

```diff
@@ -44,24 +45,31 @@ package final class LoginViaEmailViewModel: ObservableObject {
     @Published var alert: Alert?
     @Published var modalDestination: LoginViaEmailSheet?
 
-    let backendInfo: BackendInfo
+    let backendEnvironment: BackendEnvironment2
     let isEmailPrefilled: Bool
     let canCreateAccount: Bool
 
-    var areProxyCredentialsRequired: Bool {
-        backendInfo.backendConfig.proxySettings?.needsAuthentication == true
+    var authenticatedProxyServerName: String? {
+        guard
+            let proxyConfig = backendEnvironment.config.proxyConfig,
+            proxyConfig.needsAuthentication
+        else {
+            return nil
+        }
+
+        return proxyConfig.host
     }
 
-    var proxyServer: String {
-        backendInfo.backendConfig.endpoints.backendURL.absoluteString
```

Comment:
This was wrong (it's the backend rest api endpoint), I believe it should be the proxy host (right @KaterinaWire ?)